client/src/Hooks/usePostsFetch.js

```diff
@@ -1,16 +1,16 @@
 import { useState, useEffect } from 'react';
-export default function useSinglePostFetch(id){
+export default function useSinglePostFetch (id) {
```

Comment:
Just the small comment here: I'd suggest to use camelCase naming for both files and folders.